.github/workflows/ci_cron_weekly.yml

```diff
@@ -171,10 +171,9 @@ jobs:
 
   test_arm64:
 
-    # Native arm64 testing - we keep this in the weekly cron as we need to pay for it so we should
-    # minimize how many jobs we do.
+    # Native arm64 testing - we keep this in the weekly cron to avoid exceeding free quota.
```

Comment:
I mean we have never exceeded the free quota on x86 jobs ;)

Seriously tho we could make one of the main jobs aarch64.

Reply:
I think moving it to CI proper would be much simpler if I wait for OpenAstronomy?

* https://github.com/OpenAstronomy/github-actions-workflows/issues/262

I can open follow-up issue here. Thanks!

Reply:
Re: quota -- since it is still in "public beta," I am playing safe for now...